src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxNetworkInterface.cs

```diff
@@ -31,12 +31,24 @@ internal LinuxNetworkInterfaceSystemProperties()
             {
                 if (File.Exists(NetworkFiles.Ipv4RouteFile))
                 {
-                    IPv4Routes = File.ReadAllLines(NetworkFiles.Ipv4RouteFile);
+                    try
+                    {
+                        IPv4Routes = File.ReadAllLines(NetworkFiles.Ipv4RouteFile);
+                    }
+                    catch (UnauthorizedAccessException)
+                    {
+                    }
                 }
 
                 if (File.Exists(NetworkFiles.Ipv6RouteFile))
                 {
-                    IPv6Routes = File.ReadAllLines(NetworkFiles.Ipv6RouteFile);
+                    try
+                    {
+                        IPv6Routes = File.ReadAllLines(NetworkFiles.Ipv6RouteFile);
+                    }
+                    catch (UnauthorizedAccessException)
+                    {
+                    }
```

Comment:
line 70 too?
```c#
catch (Exception e) when (e is FileNotFoundException || e is UnauthorizedAccessException)
```

Reply:
I don't see anything related on line 70 ...
We already test for existence of the file, no need to change that into an exception IMO.

Reply:
ops, it was line 60 for EtcResolvConfFile.

Reply:
Yeah, I was considering that one earlier and then I forgot about it. Given that we can already live without the file, we can probably extend it also to inaccessible file ...

Reply:
Thanks for pointing it out @kasperk81! The PR is now updated.